spec/shared_selenium_session.rb

```diff
@@ -234,6 +234,19 @@
       end
     end
 
+    describe '#active_element', requires: [:js] do
```

Comment:
Lets move this into the non-selenium specific specs - something like lib/capybara/spec/session/active_element_spec.rb

Reply:
Also, please add relevant tabindex attributes in the form since the test depends on the tabbing order.

Reply:
The current tests assertions are about the first field that is not marked with `[disabled]`.

> Also, please add relevant tabindex attributes in the form since the test depends on the tabbing order.

What did you have in mind?